Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -1048,4 +1049,24 @@ BOOST_PYTHON_MODULE(rdMolDescriptors) {
   python::def("CalcNumBridgeheadAtoms", numBridgeheadAtoms,
               (python::arg("mol"), python::arg("atoms") = python::object()),
               docString.c_str());
+
```

Comment:
Yeah, I wanted to verify that this was a good path forward before moving to Python.  I want users to be able to register Python/Java property calculators as well which will take some effort.